### PR TITLE
🏗 Remove gcloud setup step from CircleCI

### PR DESCRIPTION
**Pull request by danielrozenberg**
Followup on #33917
To be merged next time we have a more serious change to be made in .circleci/config.yml, to minimize build disruptions

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -77,12 +77,6 @@ commands:
           root: /tmp
           paths:
             - workspace
-  setup_gcloud_storage:
-    # TODO(danielrozenberg): remove this after deploy-bot can use CircleCI artifacts.
-    steps:
-      - run:
-          name: 'Setup Google Cloud Storage'
-          command: ./.circleci/setup_storage.sh
```
Comment (rsimha) [inline .circleci/config.yml:85]: I assume you also meant to delete this file?

**Review by rsimha — APPROVED** [review]
LGTM. Thanks for the thorough, step-by-step transition!

File: .circleci/config.yml (hunk 1 above)
Comment (danielrozenberg) [reply .circleci/config.yml:85]: Correct. Done.

**Comment by rsimha** [comment]
Merging this along with #34169.

**Comment by wayneglicc** [comment]
Duplicate of [#[](url)]([url]([url](url)))

**Pull request merged**